Rollbar/RollbarResponse.cs

```diff
@@ -27,6 +27,15 @@ public class RollbarResponse
         /// </value>
         public RollbarResult Result { get; set; }
 
+        /// <summary>
+        /// Gets or sets the HTTP details.
+        /// </summary>
+        /// <value>
+        /// The HTTP details.
+        /// </value>
+        [JsonIgnore]
+        public string HttpDetails { get; set; }
```

Comment:
What is the point of this string? Is it just for debugging?

Reply:
yes, it provides you with http request-response strings that were exchanged with the server.